resolwe/flow/views/relation.py

```diff
@@ -48,9 +48,9 @@ def _filter_queryset(self, queryset):
 
         id_ = query_params.get('id', None)
         relation_type = query_params.get('type', None)
-        label = query_params.get('label', None)
+        category = query_params.get('category', None)
```

Comment:
Don't forget to update the function docstring.

Reply:
I suspect we will have to rethink the filtering soon. @JureZmrzlikar, @hadalin Think what you need for your use cases.